gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactor.java

```diff
@@ -105,6 +109,10 @@
   public static final String COMPACTION_DEST_SUBDIR = COMPACTION_PREFIX + "dest.subdir";
   public static final String DEFAULT_COMPACTION_DEST_SUBDIR = "daily";
 
+  // The subdir name of output dataset paths containing late events, e.g., "daily_late" in "/data/input/PasswordChangeEvent/daily_late/2015/09/06".
+  public static final String COMPACTION_DEST_LATE_SUBDIR = COMPACTION_PREFIX + "dest.late.subdir";
+  public static final String DEFAULT_COMPACTION_DEST_LATE_SUBDIR = "daily_late";
```

Comment:
You can just use `COMPACTION_DEST_SUBDIR` + `_late`, so that there's one less property the user needs to specify.